autoware_lanelet2_extension/lib/query.cpp

```diff
@@ -50,64 +50,27 @@
 
 using lanelet::utils::to2D;
 
-namespace lanelet::utils
+namespace lanelet::utils::query
```

Comment:
This change seems to be a refactoring that is unrelated to the `format_version`. If it's true, I think it should not be included in this PR to clarify the modifications made in this PR.